pkg/diagnose/connectivity/trace_request.go

```diff
@@ -126,3 +131,22 @@ func verifyEndpointResponse(writer io.Writer, statusCode int, responseBody []byt
 	}
 	fmt.Fprintf(writer, "Received status code %v from the endpoint ====> %v\n", statusCode, scrubber.ScrubLine(statusString))
 }
+
+// noResponseHints aims to give hints when the endpoint did not respond.
+// For instance, when sending an HTTP request to a HAProxy endpoint configured for HTTPS
+// the endpoint send an empty response. As the error 'EOF' is not very informative, it can
+// be interesting to 'wrap' this error to display more context.
+func noResponseHints(writer io.Writer, err error) {
+	endpoint := config.GetMainInfraEndpoint()
+	parsedURL, parseErr := url.Parse(endpoint)
+	if parseErr != nil {
+		fmt.Fprintf(writer, "Could not parse url '%v' : %v", scrubber.ScrubLine(endpoint), parseErr)
```

Comment:
```suggestion
		fmt.Fprintf(writer, "Could not parse url '%v' : %v ====> %v\n", scrubber.ScrubLine(endpoint), parseErr, color.RedString("FAIL")))
```
Just wondering if for consistency the log message should change to this, WDYT?

Reply:
I added `scrubber.ScrubLine` here and in the hooks but the `color.RedString("FAIL")` is redundant because it will be printed before